packages/tokens/src/colors.ts

```diff
@@ -0,0 +1,300 @@
+import { palette } from '@leafygreen-ui/palette';
+
+import { Mode } from './mode';
+
+const { black, blue, gray, green, red, white, yellow } = palette;
+
+const color = {
+  [Mode.Dark]: {
+    bg: {
```

Comment:
I know it's kindof a long word, but I think this should be `background`. We'll get the autocomplete either way, and the full word makes it much clearer

Reply:
that's fine i'll update